pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>ldap</artifactId>
-  <version>1.20</version>
+  <version>1.21-SNAPSHOT</version>
```

Comment:
This change makes me think you have checked out from the tag instead of from master. Please, take care and rebase just in case.

Reply:
solved